sampleapps/consumer/rbusMethodConsumer.c

```diff
@@ -109,8 +109,6 @@ int main(int argc, char *argv[])
         value2=rbusObject_GetValue(outParams, "error_string");
```

Comment:
Hi,
Fix to remove these rbusValue release is good but the the value itself is not needed right.
Can you just remove both value1 and value2 and use `rbusObject_fwrite(outParams, 1, stdout);`  and rbusObject_Release()..

Thanks,